modules/dlscript/dl_script.cpp

```diff
@@ -176,36 +176,84 @@ Error DLScript::reload(bool p_keep_state) {
 bool DLScript::has_method(const StringName &p_method) const {
 	if (!script_data)
 		return false;
-	return script_data->methods.has(p_method);
+	DLScriptData *data = script_data;
+
+	while (data) {
+		if (data->methods.has(p_method))
+			return true;
+
+		data = data->base_data;
+	}
+
+	return false;
 }
 
 MethodInfo DLScript::get_method_info(const StringName &p_method) const {
 	if (!script_data)
 		return MethodInfo();
+	DLScriptData *data = script_data;
+
+	while (data) {
+		if (data->methods.has(p_method))
+			return data->methods[p_method].info;
+
+		data = data->base_data;
+	}
+
 	ERR_FAIL_COND_V(!script_data->methods.has(p_method), MethodInfo());
-	return script_data->methods[p_method].info;
+	return MethodInfo();
```

Comment:
The condition above is always true because if it wasn't then the function would've returned in the loop.

This last return is just to keep the compiler happy because it doesn't know better.